ovos_plugin_common_play/ocp/base.py

```diff
@@ -25,9 +25,15 @@ def player(self):
 
     @property
     def settings(self):
-        if not self._player:
-            return OCPSettings()
-        return self._player.settings
+        if self._player:
+            return self._player.settings
+
+        default_path = join(get_xdg_config_save_path(), 'apps',
+                            'ovos_common_play', 'settings.json')
```

Comment:
do we have a reference to self.skill_id instead of hardcoding the string "ovos_common_play"?

Reply:
I don't think so.. if we had `self._player`, we'd use that directly.. I could put it in a class variable if we wanted to support changing this?

Reply:
not necessarily support changing it, just wanted to maintain the string definition in a single place, maybe a gloval OCP_ID variable?